examples/traverse-ipld-graphs/git.js

```diff
@@ -28,46 +26,35 @@ createNode((err, ipfs) => {
     path.join(__dirname, '/git-objects/ee71cef5001b84b0314438f76cf0acd338a2fd21')
   ]
 
-  asyncEach(gitObjects, (gitObjectsPath, cb) => {
-    const data = fs.readFileSync(gitObjectsPath)
-
-    multihashing(data, 'sha1', (err, multihash) => {
-      if (err) {
-        return cb(err)
-      }
-      const cid = new CID(1, 'git-raw', multihash)
-      console.log(cid.toBaseEncodedString())
+  await Promise.all(gitObjects.map(async gitObjectsPath => {
+    const data = await fs.readFile(gitObjectsPath)
+    const multihash = await promisify(multihashing)(data, 'sha1')
```

Comment:
Same as previously a newer version of `multihashing-async` wouldn't need a promisify.